### DISCUSSION TITLE
Question/Possible Spoiler

### DISCUSSION THREAD

**Discussion by scohn** [Monika-After-Story/MonikaModDev#8787] [appreciation]
First off, the new Halloween update is awesome. Big respect to everyone who worked so hard on it.
I kinda have a bit of a question/request. I know there's a second costume where she has some cat ears or something. I was wondering if it's possible for her to change to that between opening and closing the game while it's still halloween. If not, could someone show a picture, I'm really curious as to what she looks like in it. If not, that's fine this update has been really cool anyway.

Thanks again to everyone for their hard work.

**Comment by Legendkiller21** [comment]
The costume is determined on the first session of halloween and there is a 10% chance for Monika to dress as "Rin". Someone posted a picture of both cgs [here](https://old.reddit.com/r/MASFandom/comments/9s6x74/i_am_actually_confused_as_hell_right_now_both_are/). Just in case, **SPOILERS IN THE LINK**, DO NOT CLICK IF YOU HAVE YET TO SEE ANY OF THE COSTUMES.

**Comment by scohn** [comment]
Thanks so much for that it looks so good.
